2024-02-23 19:25:31 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trade-stocks
2024-02-23 19:25:31 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 301, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55fd8c3ca303 <unknown>
#1 0x55fd8c0af2b7 <unknown>
#2 0x55fd8c0facc0 <unknown>
#3 0x55fd8c0fadb1 <unknown>
#4 0x55fd8c13eb44 <unknown>
#5 0x55fd8c11d3ad <unknown>
#6 0x55fd8c13c07c <unknown>
#7 0x55fd8c11d123 <unknown>
#8 0x55fd8c0ed095 <unknown>
#9 0x55fd8c0ee09e <unknown>
#10 0x55fd8c38e6ab <unknown>
#11 0x55fd8c3924ba <unknown>
#12 0x55fd8c37af85 <unknown>
#13 0x55fd8c39312f <unknown>
#14 0x55fd8c35ee6f <unknown>
#15 0x55fd8c3b75f8 <unknown>
#16 0x55fd8c3b77c2 <unknown>
#17 0x55fd8c3c94a4 <unknown>
#18 0x7f25ab494ac3 <unknown>

2024-02-23 19:25:51 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trading/platform/charting/3644116541985988?side=buy&ids=27342779713672388,22855552501634244,3644116541985988,27041281599427780,422839530378436,5983666832495812,427748677997764,2604592722433220,16150730595456196,21621659937100996,27045129890124996,21659404109698244,93810675766468,1496662958757060,123763777688772,1679716659553719492,145998823326404,10384900209333444,408279591244996,356886012581060,53137335473348,283145719076036,232327666029764,15839732013552836,508931114652423506,374675767075865,1196668083066052,21680337780298948,3586297714002,194965745456466,609885377874,1717986940242,39101382284626,287256002712914,336240104723794,426090820621508
2024-02-23 19:25:51 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 301, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55fd8c3ca303 <unknown>
#1 0x55fd8c0af2b7 <unknown>
#2 0x55fd8c0facc0 <unknown>
#3 0x55fd8c0fadb1 <unknown>
#4 0x55fd8c13eb44 <unknown>
#5 0x55fd8c11d3ad <unknown>
#6 0x55fd8c13c07c <unknown>
#7 0x55fd8c11d123 <unknown>
#8 0x55fd8c0ed095 <unknown>
#9 0x55fd8c0ee09e <unknown>
#10 0x55fd8c38e6ab <unknown>
#11 0x55fd8c3924ba <unknown>
#12 0x55fd8c37af85 <unknown>
#13 0x55fd8c39312f <unknown>
#14 0x55fd8c35ee6f <unknown>
#15 0x55fd8c3b75f8 <unknown>
#16 0x55fd8c3b77c2 <unknown>
#17 0x55fd8c3c94a4 <unknown>
#18 0x7f25ab494ac3 <unknown>